src/util/ssl_utils.js

```diff
@@ -44,6 +44,7 @@ const certs = {
     MGMT: new CertInfo('/etc/mgmt-secret'),
     S3: new CertInfo('/etc/s3-secret'),
     EXTERNAL_DB: new CertInfo('/etc/external-db-secret'),
+    STS: new CertInfo('/etc/sts-secret'),
```

Comment:
As long as we add code here, I suggest moving the certs paths to `config.js`, so it will be easier to override if necessary. 
```suggestion
    STS: new CertInfo(config.STS_SECRET_PATH),
```